src/engine/input/key_mapping/InputEventItem.cs

```diff
@@ -260,6 +263,12 @@ private void OnKeybindingSuccessfullyChanged()
 
         // Update the button text
         UpdateButtonText();
+
+        // Consume current input even so it is only used for rebinding
```

Comment:
Typo in `event`?

Reply:
I think the Input being consumed in OnKeybindingSuccessfullyChanged is okay.
I'm just not sure if it should be consumed on other logical paths as well.